src/MCPClient/lib/settings/common.py

```diff
@@ -127,6 +130,13 @@
     'fpr',
 )
 
+# Configure logging manually
+LOGGING_CONFIG = None
+
+# Location of the logging configuration file that we're going to pass to
+# `logging.config.fileConfig` unless it doesn't exist.
+LOGGING_CONFIG_FILE = '/etc/archivematica/clientConfig.logging.json'
```

Comment:
given this convention, which Archivematica is following, of looking for config in /etc/archivematica, I suggest adding a readme.md at src/MCPClient/install/readme.md that describes what to do with the contents of install (and same thing in src/dashboard/install/.

Does that make sense?  Those readme's would become the documentation for how to set the logging config